apps/cow-fi/app/(main)/cow-swap/page.tsx

```diff
@@ -303,10 +303,10 @@ export default function Page() {
             >
               <TopicCardInner contentAlign="left">
                 <TopicTitle color={`var(${UI.COLOR_NEUTRAL_100})`} fontSize={51}>
-                  $2,500
+                  $67k
```

Comment:
Average trade size went to 2.5k to 67k?

Reply:
That's why I asked @cowmarketing to verify the query sources/numbers.